addons/mail/static/src/views/web/list/archive_disabled_list_controller.js

```diff
@@ -0,0 +1,10 @@
+/** @odoo-module */
+
+import { ListController } from "@web/views/list/list_controller";
+
+export class ArchiveDisabledListController extends ListController {
```

Comment:
Maybe not if we accept that the user archive activities but archiving them is not the same as “mark as done” because no message is added to the record thread. And it will have a behavior different when “keep done” is enabled or not (when enabled, it will be considered as done), when not enabled it won’t be deleted and will reappears when activating “keep done”. It is not a big problem so maybe we can rid of it. Unfortunately, I haven’t found another way to disable that feature (archive=”false”, or setting active field as readonly). What do you think ? Do we remove it ?